### ISSUE TITLE
Clock Demo Not Flashing Separator

### ISSUE THREAD

**Issue by joeyhoer** [tidbyt/pixlet#91]
I copied the clock demo code exactly (below):

```
load("render.star", "render")
load("time.star", "time")

def main(config):
    timezone = config.get("timezone") or "America/New_York"
    now = time.now().in_location(timezone)

    return render.Root(
        delay = 500,
        child = render.Box(
            child = render.Animation(
                children = [
                    render.Text(
                        content = now.format("3:04 PM"),
                        font = "6x13",
                    ),
                    render.Text(
                        content = now.format("3 04 PM"),
                        font = "6x13",
                    ),
                ],
            ),
        ),
    )
```

When rendering/serving this as a `webp` on my Mac, I see the separator flashing. However, when I push this to the Tidbyt via the API, the separator does not flash. Is anyone else able to reproduce this?

**Comment by tavdog** [comment]
I am seeing this same problem.  Displays fine in the browser but is not animating on the tidbyt.

**Comment by tavdog** [comment]
Here is the hacky workaround https://discuss.tidbyt.com/t/entire-webp-file-loop-not-publishing-to-tidbyt/1336/2

**Comment by betterengineering** [comment]
Hey folks! Thanks for the issue and so sorry for the trouble. I am able to reproduce this issue as well. I took a look and `pixlet` is doing what it's supposed to. This command will render the clock and open it in a browser (Chrome has the best WebP support):
```
pixlet render examples/clock.star
open -a 'Google Chrome' examples/clock.webp
```

I dug a bit deeper and found the issue is with our API. I've identified the line that needs changed and will have it fixed shortly.
